boards: up_squared: add timeout multiplier

arikgreen opened the pull request:
To accommodate for some slow tests on up2.
For example: tests/arch/x86/info/arch.x86.info.userspace
Execution time of this test is close to 55s and sometimes above up to the 60s.

Fixes https://github.com/zephyrproject-rtos/zephyr/issues/80134

nashif commented:
why does this behave differently on 3.7 branch?

arikgreen commented:
> why does this behave differently on 3.7 branch?

up2 behaves same as on main but on main it has set timeout_multiplier so we don't see failes
Execution time on main:
date | test suite name | test case name | tc status | tc execution   time
-- | -- | -- | -- | --
Oct,   12 2024 | tests/arch/x86/info/arch.x86.info | arch.x86.info | passed | 56.94
  | tests/arch/x86/info/arch.x86.info.userspace | arch.x86.info.userspace | passed | 56.29
Oct,   23 2024 | tests/arch/x86/info/arch.x86.info | arch.x86.info | passed | 65.58
  | tests/arch/x86/info/arch.x86.info.userspace | arch.x86.info.userspace | passed | 56.65
Oct,   22 2024 | tests/arch/x86/info/arch.x86.info | arch.x86.info | passed | 63.02
  | tests/arch/x86/info/arch.x86.info.userspace | arch.x86.info.userspace | passed | 63.52
Oct, 21 2024 | tests/arch/x86/info/arch.x86.info | arch.x86.info | passed | 55.77
  | tests/arch/x86/info/arch.x86.info.userspace | arch.x86.info.userspace | passed | 55.75
Oct, 20 2024 | tests/arch/x86/info/arch.x86.info | arch.x86.info | passed | 55.58
  | tests/arch/x86/info/arch.x86.info.userspace | arch.x86.info.userspace | passed | 64.62

nashif commented:
> up2 behaves same as on main but on main it has set timeout_multiplier so we don't see failes
> Execution time on main:

does it? looking at https://github.com/zephyrproject-rtos/zephyr/blob/main/boards/up-bridge-the-gap/up_squared/up_squared.yaml it does not have a multiplier

nashif commented:
@arikgreen you need to submit this to main, and label it for backporting by the bot.

arikgreen committed:
boards: Fix to accommodate for slowly tests on UP2

To accommodate for some slow tests on up2.
For example: tests/arch/x86/info/arch.x86.info.userspace
Execution time of this test is close to 55s and sometimes
above up to the 60s.
Related issue: #80134

Signed-off-by: Artur Wilczak <arturx.wilczak@intel.com>
